public/app/features/dashboard/state/PanelQueryRunner.test.ts

```diff
@@ -1,10 +1,18 @@
-import { PanelQueryRunner } from './PanelQueryRunner';
-import { PanelData, DataQueryRequest, dateTime, ScopedVars } from '@grafana/data';
+import { PanelConfig, PanelQueryRunner } from './PanelQueryRunner';
+// Importing this way to be able to spy on grafana/data
+import * as grafanaData from '@grafana/data';
```

Comment:
what do you want to spy on? this makes the file very strange (especially for the types), I would import the types normally

Reply:
see https://github.com/grafana/grafana/blob/98bd8196400eb4ad7b35f65065ca779037eaeec1/public/app/features/dashboard/state/PanelQueryRunner.test.ts#L231 for example. But you are right about types